src/main/resources/locales/en/iac-daily-list.json

```diff
@@ -18,7 +18,7 @@
     "respondent": "Respondent",
     "interpreterLanguage": "Interpreter Language",
```

Comment:
The 'apellant' above needs to be'Appellant/Applicant. Same for Welsh translation

Reply:
Added "Apelydd/Ymgeisydd" for the Welsh language. "Appellant/Applicant", for the English language. Thanks Chris.